sql/src/main/java/io/crate/analyze/relations/QueriedRelation.java

```diff
@@ -26,4 +26,7 @@
 public interface QueriedRelation extends AnalyzedRelation {
 
     QuerySpec querySpec();
+
+    // relation Id used to distinguish between relations in the Fetch phase (union)
```

Comment:
I'd document what it is instead of how it is used.

E.g. : Unique id within a tree of relations.